template/src/core/network/fetcherUtils.ts

```diff
@@ -0,0 +1,10 @@
+function fetchJSONMiddleware(response: globalThis.Response) {
+    return response.json().catch((_error) => ({}));
+  }
+  
+  function stringifyJSON(value: any) {

```

Comment:
It looks like it isn't used anywhere, maybe we can remove it 💡

Reply:
Thanks for your review! I didn't add this function as `bodyParser` to our Fetcher instance in this example but maybe i should have. We need to use this function when we create `Fetcher` instance if we'll send POST requests with object payloads. Here is the example use case:

```tsx
const HipoApi = new Fetcher({
  baseUrl: process.env.REACT_APP_API_BASE_URL!,
  initOptions: {
    headers: {
      "Content-Type": "application/json",
      "x-csrftoken": csrftoken || ""
    },
    credentials: "include"
  },
  bodyParser: stringifyJSON, // <--
  responseMiddlewares: [fetchJSONMiddleware]
});
```

Reply:
Can't we simply use JSON.stringify here?
```ts
bodyParser: JSON.stringify
```

I believe creating a function that just calls a single function is redundant.

Reply:
@jamcry  Makes sense, thank you! I'll remove it